morgoth/addons/models.py

```diff
@@ -21,9 +22,78 @@
 )
 
 
+class VersionNumber(object):
```

Comment:
product_details has a class that you might be able to subclass or use instead: https://github.com/mozilla/django-product-details/blob/master/product_details/version_compare/__init__.py#L23